docs/index.md

```diff
@@ -124,11 +135,12 @@ Now if we run `docker-compose up`, it'll pull a Redis image, build an image for
     redis_1 | [8] 02 Jan 18:43:35.576 # Server started, Redis version 2.8.3
     web_1   |  * Running on http://0.0.0.0:5000/
 
-The web app should now be listening on port 5000 on your docker daemon (if you're
-using boot2docker, `boot2docker ip` will tell you its address).
+The web app should now be listening on port 5000 on your docker daemon (if
```

Comment:
on your Docker daemon host

caps, and the daemon doesn't have a port to listen on, its the host